rpc/RpcReportPlayer.cpp

```diff
@@ -0,0 +1,14 @@
+#include "il2cpp-appdata.h"
+#include "utility.h"
+#include "_rpc.h"
+
+RpcReportPlayer::RpcReportPlayer(PlayerSelection selected_player)
+{
+	this->reportedPlayer = selected_player;
+}
+
+void RpcReportPlayer::Process()
+{
+	if (reportedPlayer.has_value())
+		PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, reportedPlayer.get_PlayerData(), NULL);
```

Comment:
We should make a small change here and check to see if the reportedPlayer is the Local Player, if so call PlayerControl_CmdReportDeadBody with NULL as the second parameter. With the current implementation if you call a meeting you end up actually reporting your own body, this is kind of funny to be honest but it should be addressed.

Reply:
That was on purpose, but understandable. It's pretty funny reporting random bodies that aren't dead.  Anticheat would probably look for it on a future update.

Reply:
Committed a safeguard to the RPC function to prevent reporting yourself.  Do note that the players tab already had a check to remove the Report Body button if you select yourself.